cast/tests/data/contracts/balance/src/lib.cairo

```diff
@@ -0,0 +1,18 @@
+#[contract]
```

Comment:
It's a nitpick, but is the contract's name somehow imposed by scarb or something? if not, let's maybe rename it to something more meaningful?

Reply:
by contract name I meant both the name ( ( ͡° ͜ʖ ͡°) ) and the file name